satori-astro fails on Cloudflare Pages due to Node.js built-in module dependency

I'm encountering a build failure when trying to use satori-astro on Cloudflare Pages. The error suggests that there's an issue with bundling the Node.js built-in "fs" module, which is being imported from the @resvg/resvg-js package, which is a dependency of satori-astro (
```
  satori-astro@0.0.1(astro@4.15.0(@types/node@22.5.1)(rollup@4.21.1)(typescript@5.5.4)):
    dependencies:
      '@resvg/resvg-js': 2.6.2
```
).

Error message:
```
[ERROR] [vite] x Build failed in 4.36s
[commonjs--resolver] [plugin vite:resolve] Cannot bundle Node.js built-in "fs" imported from "node_modules\.pnpm\@resvg+resvg-js@2.6.2\node_modules\@resvg\resvg-js\js-binding.js". Consider disabling ssr.noExternal or remove the built-in dependency.
```

This error occurs during the build process and prevents the successful deployment on Cloudflare Pages.

My astro config file:
```
import { defineConfig, envField } from "astro/config";

import cloudflare from "@astrojs/cloudflare";
import svelte from "@astrojs/svelte";
import auth from "auth-astro";
import tailwind from "@astrojs/tailwind";
import icon from "astro-icon";
import db from "@astrojs/db";
import i18n from "@astrolicious/i18n";
import satori from "satori-astro";

// https://astro.build/config
export default defineConfig({
  site: "xxxxxx",
  prefetch: true,
  integrations: [
    icon({
      include: {
        "material-symbols": ["translate-rounded", "dark-mode-rounded", "light-mode-rounded", "send-rounded", "info-rounded", "brightness-auto-rounded", "borg-rounded"],
        bxl: ["google"],
      },
    }),
    i18n({
      defaultLocale: "en",
      locales: ["en", "it", "es", "fr", "de"],
      client: {
        data: true,
        paths: true,
        translations: true,
      },
      sitemap: true,
    }),
    svelte(),
    auth(),
    tailwind(),
    db(),
    satori(),
  ],
  output: "server",
  adapter: cloudflare(),
  security: {
    checkOrigin: true,
  },
  vite: {
    ssr: {
      external: ["buffer", "path", "fs", "os", "async_hooks"].map((i) => `node:${i}`),
    },
  },
  experimental: {
    contentIntellisense: true,
    env: {
      schema: {
xxxx
      },
    },
  },
});
```

On Cloudflare Pages I set the compatibility flags to nodejs_compat, but it didn't change anything. Please advise since this is preventing me from pushing a SEO update. Thank you for your work anyways, it works perfectly in dev, its the build that is causing problems.

--- a/.changeset/twenty-mirrors-watch.md
+++ b/.changeset/twenty-mirrors-watch.md
@@ -1,0 +1,5 @@
+---
+"satori-astro": minor
+---
+
+Updates internals to use `wasm` for Cloudflare compatibility

--- a/package/package.json
+++ b/package/package.json
@@ -28,7 +28,7 @@
     "astro": "^4.0.0"
   },
   "dependencies": {
-    "@resvg/resvg-js": "^2.6.0",
+    "@resvg/resvg-wasm": "^2.6.2",
     "astro-integration-kit": "^0.0.9",
     "satori": "^0.10.11"
   }

--- a/package/src/og.ts
+++ b/package/src/og.ts
@@ -1,4 +1,4 @@
-import { Resvg } from "@resvg/resvg-js";
+import { Resvg, initWasm } from "@resvg/resvg-wasm";
 import satori from "satori";
 import type {
 	SatoriAstroOGOptions,
@@ -25,6 +25,7 @@ export const satoriAstroOG = ({
 					? _resvgOptions({ width, height })
 					: _resvgOptions;
 
+			await initWasm(undefined as any).catch(() => {});
 			return new Resvg(await this.toSvg(satoriOptions), {
 				fitTo: { mode: "width", value: width },
 				...resvgOptions,

--- a/playground/package.json
+++ b/playground/package.json
@@ -14,7 +14,6 @@
     "@astrojs/tailwind": "^5.1.0",
     "@resvg/resvg-js": "^2.6.0",
     "astro": "^4.2.6",
-    "astro-carton": "^0.1.1",
     "satori-astro": "workspace:*",
     "satori-html": "^0.3.2",
     "tailwindcss": "^3.4.1"

--- a/playground/src/components/OG.astro
+++ b/playground/src/components/OG.astro
@@ -1,3 +1,0 @@
-<div style="display: flex; justify-items: center; align-items: center; background-color: red; font-family: Inter; height: 100%;">
-    <h1 style="color: blue;">Test toResponse</h1>
-</div>

--- a/playground/src/pages/test-to-response.png.ts
+++ b/playground/src/pages/test-to-response.png.ts
@@ -1,9 +1,6 @@
 import type { APIRoute } from "astro";
 import { satoriAstroOG } from "satori-astro";
 import { html } from "satori-html";
-// import "astro-carton";
-// import { AstroCarton } from "astro-carton/runtime";
-import OG from "../components/OG.astro";
 
 export const GET: APIRoute = async () => {
 	const fontFile = await fetch(
@@ -11,29 +8,10 @@ export const GET: APIRoute = async () => {
 	);
 	const fontData: ArrayBuffer = await fontFile.arrayBuffer();
 
-	await import("astro-carton")
-	const { AstroCarton } = await import("astro-carton/runtime")
-
-	// return await satoriAstroOG({
-	// 	template: html`<div style="display: flex; justify-items: center; align-items: center; background-color: red; font-family: Inter; height: 100%;">
-    //                         <h1 style="color: blue;">Test toResponse</h1>
-    //                     </div>`,
-	// 	width: 1920,
-	// 	height: 1080,
-	// }).toResponse({
-	// 	satori: {
-	// 		fonts: [
-	// 			{
-	// 				name: "Inter Latin",
-	// 				data: fontData,
-	// 				style: "normal",
-	// 			},
-	// 		],
-	// 	},
-	// });
-	const carton = new AstroCarton();
 	return await satoriAstroOG({
-		template: html(await carton.renderToString(OG)),
+		template: html`<div style="display: flex; justify-items: center; align-items: center; background-color: red; font-family: Inter; height: 100%;">
+                            <h1 style="color: blue;">Test toResponse</h1>
+                        </div>`,
 		width: 1920,
 		height: 1080,
 	}).toResponse({

